CommandLine/libEGM/gmk.cpp

```diff
@@ -44,17 +48,26 @@ ostream err(nullptr);
 static vector<std::string> tempFilesCreated;
 static bool atexit_tempdata_cleanup_registered = false;
 static void atexit_tempdata_cleanup() {
-  for (std::string &tempFile : tempFilesCreated)
-    unlink(tempFile.c_str());
+  for (const std::string &tempFile : tempFilesCreated)
+    DeleteFile(tempFile);
 }
 
 std::string writeTempDataFile(char *bytes, size_t length) {
-  char temp[] = "gmk_data.XXXXXX";
-  int fd = mkstemp(temp);
-  if (fd == -1) return "";
-  write(fd, bytes, length);
-  close(fd);
-  return temp;
+  static int increment = 0;
```

Comment:
Why did you bother doing all the work to extract our generalized filesystem logic from EGM, only to ignore `std::filesystem::temp_directory_path` / `boost::filesystem::unique_path`?

Reply:
Addressed: 9df6bd2b608ef5dd59b1c69583c75ddf47fc9f5d